napari/_qt/widgets/qt_viewer_dock_widget.py

```diff
@@ -59,6 +59,9 @@ class QtViewerDockWidget(QDockWidget):
         them distribute across the vertical space).  By default, True.
     """
 
+    closed = Signal()
```

Comment:
really don't like creating new signals and involving other widgets (here the QMainWindow) in the business of "managing" other widgets if it's at all possible to have that other widget manage itself.  In this case, it looks like you're creating new signals so that you can check and uncheck the menu action when this gets hidden and shown... but don't you already have a handle to that action below (around line 275)?  do we need the main window to do this?